gateway/src/main/scala/filodb/timeseries/TestTimeseriesProducer.scala

```diff
@@ -97,20 +97,20 @@ object TestTimeseriesProducer extends StrictLogging {
       val endQuery = startQuery + 300
       val query =
         s"""./filo-cli '-Dakka.remote.netty.tcp.hostname=127.0.0.1' --host 127.0.0.1 --dataset prometheus """ +
-        s"""--promql 'heap_usage{dc="DC0",_ns="App-0"}' --start $startQuery --end $endQuery --limit 15"""
+        s"""--promql 'heap_usage{_ns="App-0"}' --start $startQuery --end $endQuery --limit 15"""
```

Comment:
Please do not update the TimeSeriesProducer as this is changing significantly in my other PR

Reply:
OK. I'll remove the changes